cpp-ch/CMakeLists.txt

```diff
@@ -46,6 +47,7 @@ add_custom_command(
         -DENABLE_TESTS=OFF
         -DENABLE_JEMALLOC=ON
         -DENABLE_MULTITARGET_CODE=ON
+        -DENABLE_UTILS=ON
```

Comment:
https://github.com/ClickHouse/ClickHouse/pull/49620 is turned off  by default, I think we should update cmake in Kyligence/ClickHouse instead of adding `-DENABLE_UTILS=ON`